pmd-java/src/main/java/net/sourceforge/pmd/lang/java/internal/JavaAstProcessor.java

```diff
@@ -49,9 +52,32 @@
     static {
         Level level;
         try {
-            level = Level.parse(System.getenv("PMD_DEBUG_LEVEL"));
+            switch (System.getenv("PMD_DEBUG_LEVEL").toLowerCase(Locale.ROOT)) {
```

Comment:
Does [`Level.toLevel(String)`](https://www.slf4j.org/api/org/apache/log4j/Level.html#toLevel-java.lang.String-) not work here?

Reply:
I've used now Level.valueOf

Reply:
Ah I see I was looking at log4j doc sorry